recipes/pysqa/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2019, Jan Janssen
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+* Redistributions of source code must retain the above copyright notice, this
```

Comment:
(This doesn't prevent this from being merged but so you know for future releases.)

This file shouldn't needed as, according to this line, there should be a copy of the licence included with every distribution of the package. It should be automatically included in the release tarballs on PyPI if you edit your `setup.cfg` to include:

```
[metadata]
license_files = LICENSE
```

See [here (and the sample project)](https://packaging.python.org/guides/distributing-packages-using-setuptools/#license-txt) for more details.

Doing this would remove the need to include this file with the recipe and also ensures that the text is always up to date.